multi-node-k8s.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+
+# get latest stable version
+# K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
+K8S_VERSION="v1.17.5"
+
+# configure cluster parameters
+WORKERS=1
+WORKER_MEM=4
+WORKER_CPU=2
+DISKS=2
+DISK_SIZE=25
+MASTER_MEM=4
+MASTER_CPU=2
+K8S_NETWORK="calico"
+# POD_NET_CIDR=192.168.123.0/24
+
+# enable additional sources for yum
+# (epel for ansible and golang)
+yum -y install epel-release
+
+# Install additional packages
+yum -y install \
+	qemu-kvm \
+	qemu-kvm-tools \
+	qemu-img \
+	libvirt \
+	ansible
+
+# install kubectl
+if [[ ! -x /usr/bin/kubectl ]]; then
+	curl -LO https://storage.googleapis.com/kubernetes-release/release/"${K8S_VERSION}"/bin/linux/amd64/kubectl
+	mv kubectl /usr/bin/kubectl
+	chmod +x /usr/bin/kubectl
+fi
+
+# Vagrant needs libvirtd running
+systemctl start libvirtd
+
+# Log the virsh capabilites so that we know the
+# environment in case something goes wrong.
+virsh capabilities
+
+# TODO: this is not the right way to install vagrant on CentOS, but SCL only
+# provides vagrant 1.x and we need >= 2.2
+if ! command -v vagrant; then
+	yum -y install https://releases.hashicorp.com/vagrant/2.2.7/vagrant_2.2.7_x86_64.rpm
+	yum -y install gcc libvirt-devel
+	vagrant plugin install vagrant-libvirt
+fi
+
+vagrant plugin install vagrant-libvirt
+
+# setup the kubernes cluster
+git clone https://github.com/galexrt/k8s-vagrant-multi-node
+cd k8s-vagrant-multi-node || exit
+
+yum install -y systemd-resolved
+systemctl start systemd-resolved
+
+# Disable SELinux
+setenforce 0
+sed -i '/^SELINUX./ { s/enforcing/disabled/; }' /etc/selinux/config
+
+# Disable swap
+swapoff -a
+sed -i '/ swap / s/^\(.*\)$/#\1/g' /etc/fstab
+
+make preflight
+make up BOX_OS=centos VAGRANT_DEFAULT_PROVIDER=libvirt KUBERNETES_VERSION="${K8S_VERSION}" MASTER_CPUS=${MASTER_CPU} NODE_CPUS=${WORKER_CPU} MASTER_MEMORY_SIZE_GB=${MASTER_MEM} NODE_COUNT=${WORKERS} NODE_MEMORY_SIZE_GB=${WORKER_MEM} DISK_COUNT=${DISKS} DISK_SIZE_GB=${DISK_SIZE} POD_NW_CIDR=192.168.123.0/24 KUBE_NETWORK=calico
```

Comment:
use variables in the top of the script for `POD_NW_CIDR=192.168.123.0/24 KUBE_NETWORK=calico`